crates/rpc_client/Cargo.toml

```diff
@@ -11,3 +11,4 @@ jsonrpc-types = "0.3.3"
 reqwest = { version = "0.11", features = ["blocking", "json"] }
 serde = "1.0.148"
 serde_json = "1.0.89"
+thiserror = "1.0.37"
```

Comment:
Out of curiosity, are you using `cargo add` to add dependencies. It's a very useful CLI tool :)

Reply:
yup! :)